integration-test/background/click-to-load-youtube.js

```diff
@@ -0,0 +1,335 @@
+            const {
+                youTubeIframeApi, youTubeStandard, youTubeNocookie
+            } = summariseYouTubeRequests(pageRequests)
+
+            expect(youTubeIframeApi.checked).toBeTrue()
+            expect(youTubeIframeApi.alwaysRedirected).toBeFalse()
+            expect(youTubeStandard.blocked).toEqual(0)
+            expect(youTubeNocookie.blocked).toEqual(0)
+            expect(youTubeNocookie.allowed).toBeGreaterThanOrEqual(1)
+        }
+
+        // When the page is reloaded, requests should be blocked again.
+        pageRequests.length = 0
+        await page.reload({ waitUntil: 'networkidle2' })
+        {
+            const {
+                youTubeIframeApi, youTubeStandard, youTubeNocookie
+            } = summariseYouTubeRequests(pageRequests)
+
+            expect(youTubeIframeApi.checked).toBeTrue()
+            expect(youTubeIframeApi.alwaysRedirected).toBeTrue()
+            expect(youTubeStandard.total).toBeGreaterThan(5)
+            expect(youTubeStandard.blocked).toEqual(youTubeStandard.total)
+            expect(youTubeStandard.allowed).toEqual(0)
+            expect(youTubeNocookie.blocked).toEqual(youTubeNocookie.total)
+            expect(youTubeNocookie.allowed).toEqual(0)
+        }
+
+        // The header button should also unblock YouTube.
+        pageRequests.length = 0
+        const headerButton = await page.evaluateHandle(
+            'document.querySelector("#short-container > div")' +
+            '.shadowRoot.querySelector("#DuckDuckGoPrivacyEssentialsCTLElementTitleTextButton")'
+        )
+        await headerButton.click()
+        await page.waitForNetworkIdle({ idleTime: 1000 })
+        {
+            const {
+                youTubeIframeApi, youTubeStandard, youTubeNocookie
+            } = summariseYouTubeRequests(pageRequests)
+
+            expect(youTubeIframeApi.checked).toBeTrue()
+            expect(youTubeIframeApi.alwaysRedirected).toBeFalse()
+            expect(youTubeStandard.blocked).toEqual(0)
+            expect(youTubeNocookie.blocked).toEqual(0)
+            expect(youTubeNocookie.allowed).toBeGreaterThanOrEqual(1)
+        }
+
+        page.close()
+    })
+
+    it('CTL: YouTube interacting with iframe API', async () => {
+        const page = await browser.newPage()
+        await page.goto(testSite, { waitUntil: 'networkidle2' })
+
+        // Test the Iframe API controls and events function correctly, even when
+        // used with an existing video.
+        {
+            const waitForExpectedBorder = expectedBorder =>
+                page.waitForFunction(expectedBorder => (
+                    document.getElementById('existing-video')
+                        .style.border.split(' ').pop() === expectedBorder
+                ),
+                { polling: 10 }, expectedBorder)
+
+            await waitForExpectedBorder('')
+
+            const button = await page.evaluateHandle(
+                'document.querySelector("div:nth-child(7) > div")' +
```

Comment:
that selector looks fragile, can we change something on the test page so that we can use something more solid like ID CSS selector here?